src/helpers/clipboard/clipboard_cells_state.ts

```diff
@@ -505,7 +524,9 @@ export class ClipboardCellsState extends ClipboardCellsAbstractState {
     for (const row of this.cells) {
       htmlTable += "<tr>";
       for (const cell of row) {
-        const cssStyle = cssPropertiesToCss(cellStyleToCss(cell.style));
+        const cssStyle = cssPropertiesToCss(
+          cellStyleToCss(this.getters.getCellComputedStyle(cell.position))
+        );
```

Comment:
This is correct, but once again 
1) try to make different commits for different fixes. Or at least mention it in the commit message/PR message
2) Create a test when you fix something

Reply:
It must have slipped my mind...sorry about that.
I've split the commit and added a test case now :+1: 
Let me know if it needs any amendments :)